classes/classes/Scenes/Areas/GlacialRift/Amarok.as

```diff
@@ -0,0 +1,164 @@
+// By Foxwells
+// Amarok, a big ass wolf from Inuit mythology. Idek. Glacial Rift is depressingly empty
+// Since it's a big ass god wolf, it's pretty hard to beat
+// Herm, since there's no set gender for it in its mythology
+// Comes with a unique bad end if you lose to it too many times, kinda like the Phoenix
+// Also: Wolf TF! :D Because like, really, why hasn't that been implemented yet? It could be sooo coool <3
+
+package classes.Scenes.Areas.GlacialRift 
+{
+	import classes.*;
+	import classes.internals.WeightedDrop;
+	
+	public class Amarok extends Monster
+	{
+		
+		protected function amarokClaw():void {
+			//Blind
+			if(findStatusEffect(StatusEffects.Blind) >= 0) {
+				outputText("The Amarok lunges for you, attempting to slash you with one of its paws, but misses completely due to its blindness.", false);
+				combatRoundOver();
+				return;
+			}
+			//Too many things for evasion...
```

Comment:
Too many. You should either use getEvasionRoll (returns unspecified boolean) or getEvasionReason (returns which reason fired to make flavor text). This way you'll check evasion twice - first in your code, then in getEvasionRoll.